frontend/src/App.js

```diff
@@ -41,17 +41,27 @@ function App() {
 
   //add http request for command to backend
   //Backend call for command using http request
-  const getCommand = async(command, args) => {
-    console.log("drop pada function")
-
+  const getCommand = async(command, group, state) => {
     fetch('http://localhost:5000/command', {
        method: 'POST',
         headers: { 'Content-Type': 'application/json '},
-         body: JSON.stringify({ commands: [{ command: command, args: args }] })
+         body: JSON.stringify({
+          commands:
+          [
+              {
+                  command: command,
+                  args:
+                  [
+                      group: group,
+                      state: state 
+                  ]
```

Comment:
This function should just accept an argument for the command name and the arguments. That way it can be used for _any_ command and not just the DROP_PADA button. This also simplifies the function because you can make the payload:
```
{
    commands:
        [
            command: command
            args: args
        ]
}
```
Honestly it should really just accept a single argument `commands` and the caller of the function should worry about adding the commands into the `commands` object (that way they can pass multiple commands if they require)